<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpharaSettingsViewWidget</class>
 <widget class="QWidget" name="SpharaSettingsViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>105</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>0.950000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>8</number>
   </property>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="m_checkBox_activateSphara">
       <property name="text">
        <string>Activate SPHARA</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="m_comboBox_spharaSystem">
       <item>
        <property name="text">
         <string>BabyMEG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VectorView</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EEG</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_label_spharaFirst">
       <property name="text">
        <string>Inner layer:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="m_spinBox_spharaFirst">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>270</number>
       </property>
       <property name="value">
        <number>270</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_label_spharaSecond">
       <property name="text">
        <string>Outer layer:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="m_spinBox_spharaSecond">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>105</number>
       </property>
       <property name="value">
        <number>105</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
